Kaguya/Kaguya/Discord/Commands/Music/Play.cs

```diff
@@ -60,9 +62,23 @@ public async Task PlayCommand([Remainder]string search)
             }
 
             LavaPlayer player = _lavaNode.GetPlayer(Context.Guild);
-            SearchResponse searchResponse = await _lavaNode.SearchYouTubeAsync(search);
+            SearchResponse searchResult;
+            try
+            {
+                searchResult = await _lavaNode.SearchYouTubeAsync(search);
+            }
+            catch (HttpRequestException e)
+            {
+                string error = "Lavalink is not connected. Please start lavalink in " + 
```

Comment:
It's generally considered a security risk to expose your system internals like this, this message should be logged for the owner but a more generic message should be presented to the public